frontend/src/templates/Field/FieldContainer.tsx

```diff
@@ -57,14 +64,25 @@ export const FieldContainer = ({
       isInvalid={!!error}
       id={schema._id}
     >
-      <FormLabel
-        questionNumber={
-          schema.questionNumber ? `${schema.questionNumber}.` : undefined
-        }
-        description={schema.description}
+      <Grid
```

Comment:
nit: i wonder if grid is needed. Is `Wrap` sufficient? then instead of hidden and adding grid areas we can just have conditional rendering ala

```tsx
{showMyInfoBadge && <Badge ... />}
```

Reply:
as discussed, we'll keep to using grid to avoid margin hell